test/performance/go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/lvm-operator/test/performance
```

Comment:
I think we should not create a separate go module for perf. Do you see any extra benefit in doing it?